exporters/export_formatter/avro_export_formatter.py

```diff
@@ -0,0 +1,65 @@
+import json
+import six
+import io
+
+from exporters.exceptions import ConfigurationError
+from exporters.export_formatter.base_export_formatter import BaseExportFormatter
+
+
+class AvroExportFormatter(BaseExportFormatter):
+    """
+    This export formatter provides a way of exporting items in AVRO format.
+    follows the avro schema 1.8 https://avro.apache.org/docs/1.8.1/spec.html
+    We write the items on a 1 per block way.
+    This allows for streaming and keeps the files consistency even if the process
+    ends unexpectedly
+
+        - schema(string)
+            json-encoded schema for the data to use in avro formatter
+
+        - schema_path(string)
+            path to the json schema for the data to use in avro formatter.
+            Useful for large schemas if readibility of the job json file is important
+    """
+
+    supported_options = {
+        'schema_path': {'type': six.string_types, 'default': None},
+        'schema': {'type': six.string_types, 'default': None},
```

Comment:
One more nitpick: the [other existing formatter that take a `schema` option is expecting a `dict`](https://github.com/scrapinghub/exporters/blob/master/exporters/export_formatter/csv_export_formatter.py#L31), this is expecting a JSONified string.

Could you change this to support the `dict` too?
It's kind of awkward to write a JSON inside of a JSON. :)

Reply:
Oh sure, easy change. I can make `schema` take a dict.